src/acl/Checklist.cc

```diff
@@ -325,8 +325,8 @@ ACLChecklist::fastCheck()
 void
 ACLChecklist::calcImplicitAnswer()
 {
-    const auto lastAction = accessList ?
-                            accessList->lastAction() : Acl::Answer(ACCESS_DUNNO);
+    const auto& lastAction = accessList ?
```

Comment:
If my conclusions in another change request are invalidated, then this `auto&` formatting inconsistency will need to be fixed as well:

```suggestion
    const auto &lastAction = accessList ?
```